### ISSUE TITLE
dom.d.ts is missing Request and Response interfaces

### ISSUE THREAD

**Issue by mohsen1** [Microsoft/TypeScript#14237]
**TypeScript Version:**  2.2

**Code**

```ts
const r: Response;
         ~~~~~~~~ Can't find name "Response"

```

```json
// tsconfig
{
  "compilerOptions": {
    "lib": [
      "dom",
      "es2015"
    ]
  }
}
```

**Comment by mhegazy** [comment]
Are you sure this is TS 2.2? here is the lib.dom.ts from this release: https://github.com/Microsoft/TypeScript/blob/a29e8cf2406dd41aea48be0fb68cde6a2e972564/lib/lib.dom.d.ts#L9365-L9374

**Comment by mohsen1** [comment]
It was an issue in `2.2.0` but is fixed in `2.2.1`
